tests/integration/s2n_client_endpoint_handshake_test.py

```diff
@@ -69,23 +77,34 @@ def try_client_handshake(endpoint, use_corked_io):
     return 0
 
 def well_known_endpoints_test(use_corked_io):
+    arguments = []
     msg = "\n\tTesting s2n Client with Well Known Endpoints:"
     if use_corked_io:
         msg = "\n\tTesting s2n Client with Well Known Endpoints using Corked IO:"
+        arguments += ["-C"]
     print(msg)
 
     maxRetries = 5
     failed = 0
-    for endpoint in well_known_endpoints:
+    for endpoint_config in well_known_endpoints:
+        endpoint = endpoint_config["endpoint"]
+
+        if "cipher_preference_version" in endpoint_config:
+            arguments += ["-c", endpoint_config["cipher_preference_version"]]
+
+        expected_cipher = None
+        if "expected_cipher" in endpoint_config:
+            expected_cipher = endpoint_config["expected_cipher"]
```

Comment:
NP: can you just do endpoint_config.get("expected_cipher") which will default to None?

Reply:
Turns out, you can. `endpoint_config["expected_cipher"]` throws a `KeyError`, but `endpoint_config.get("expected_cipher")` defaults to `None` if the key isn't in the dict.